### PR TITLE
Add PlayJev

### PR DESCRIPTION
**Pull request by unikcc**
One project, `data/projects.json` only, category `games`. `npm run validate` passes; README and site left untouched.

PlayJev is Qwen3.5-0.8B-Base fine-tuned to play ten classic browser games (Tetris, Snake, Pacman, a racer, Space Invaders, Sokoban, Infinite Mario, Floppy Bird, Breakout, 2048) from 448 px screenshots. Where Jev fits: state plus typed options in, a probability per option out. The state is the frame, the options are the game's legal moves in the frozen OpenJev prompt, and the answer is the softmax over the option letters at a single position, one forward pass with nothing generated. It serves the same `/v1/systemone` request shape, so a demo tile can be pointed at a local server instead of the recordings. It is an open replica of the interface, not TypeSafe's model.

I built it. Weights: https://huggingface.co/OmniJev/PlayJev-0.8B. The demo runs all ten games in the browser at once.

---
<a href="https://app.blacksmith.sh/hellogumbo/codesmith/awesome-jev/pr/28?ref=codesmith_pr_footer"><picture><source media="(prefers-color-scheme: dark)" srcset="https://pr-comments-assets.blacksmith.sh/codesmith/view-with-codesmith-dark-v2.svg"><source media="(prefers-color-scheme: light)" srcset="https://pr-comments-assets.blacksmith.sh/codesmith/view-with-codesmith-light-v2.svg">View with [code]smith</picture></a> <a href="https://backend.blacksmith.sh/track/enable-autofix?expires=1792416087&installation_model_id=429846&pr_number=28&ref=codesmith_pr_footer&repository=hellogumbo%2Fawesome-jev&return_to=https%3A%2F%2Fgithub.com%2Fhellogumbo%2Fawesome-jev%2Fpull%2F28&signature=09738451b1dfe975b9dcac7e397c57110491b0eb30dc89c26cfd8a424a83d021"><picture><source media="(prefers-color-scheme: dark)" srcset="https://pr-comments-assets.blacksmith.sh/codesmith/autofix-with-codesmith-dark.svg"><source media="(prefers-color-scheme: light)" srcset="https://pr-comments-assets.blacksmith.sh/codesmith/autofix-with-codesmith-light.svg">Autofix with [code]smith</picture></a>
<sup>Need help on this PR? Tag <code>@codesmith-bot</code> with what you need. Autofix is disabled.</sup>

## Summary by CodeRabbit

- **New Features**
  - Added PlayJev to the projects listing, including its website, repository, category, language, and metadata.

### CODE REVIEW

**Comment by scald** [comment]
Landed on main as 3ea7c74 with you as the commit author, so it shows on your contributions. Live at https://awesomejev.com/#games-simulations.

Closing the PR rather than merging because the site's data file gets appended to nightly, which makes fork branches conflict faster than they can be reviewed. Landing the entry directly keeps your authorship without force-pushing to your fork. Thanks for the submission.
